downstream/modules/automation-hub/con-approval-pipeline.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+// assembly-repo-management.adoc
+
+
+[id="con-approval-pipeline"]
+
+= Approval pipeline
+
+Automation Hub allows you to approve collections into any repository that is marked with the `pipeline=approved` label. By default, Hub ships with one repository for approved content, but you have the option to add more from the repository creation screen. Repositories marked with this label are not eligible for direct publishing and collections must come from one of the staging repositories.
```

Comment:
Missing attributes (one for the start of the sentence and one within a sentence; lower case mostly) . Do we have an attribute for just "Hub"? Might be good to add that to the list of attributes.

```suggestion
{HubNameStart} allows you to approve collections into any repository that is marked with the `pipeline=approved` label. By default, {HubName} ships with one repository for approved content, but you have the option to add more from the repository creation screen. Repositories marked with this label are not eligible for direct publishing and collections must come from one of the staging repositories.
```

Reply:
Changed to attributes.